week2/Week_2 (DraftForClass).ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>Watch out for spelling.</p><blockquote>copmlexity -&gt; complexity</blockquote><blockquote>differnce -&gt; difference</blockquote><p> </p><p>I find the 'where is it going to stop?' question a little unclear. Maybe, 'how far will the boat travel from the origin?'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>